packages/react/src/primitives/Button/__tests__/Button.test.tsx

```diff
@@ -3,9 +3,17 @@ import { render, screen } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 
 import { Button } from '../Button';
+import { ButtonColorTheme } from '../../types';
 import { ComponentClassNames } from '../../shared';
 
 describe('Button test suite', () => {
+  const SUPPORTED_COLOR_THEMES = [
+    'info',
+    'success',
+    'warning',
+    'error',
+    'overlay',
+  ];
```

Comment:
Nit: declare constant at global scope

```suggestion
const SUPPORTED_COLOR_THEMES = [
  'info',
  'success',
  'warning',
  'error',
  'overlay',
];

describe('Button test suite', () => {
```